daemons/controld/controld_execd.c

```diff
@@ -2532,6 +2533,18 @@ process_lrm_event(lrm_state_t *lrm_state, lrmd_event_data_t *op,
     CRM_CHECK(op != NULL, return);
     CRM_CHECK(op->rsc_id != NULL, return);
 
+    // Remap new status codes for older DCs
+    if (compare_version(fsa_our_dc_version, "3.2.0") < 0) {
```

Comment:
This assumes we will get the feature set bump from #1804 first. We can always add the bump here if not.

Reply:
Does it make sense to convert that to symbolic names like
`FEATURE_SET_MIN_FOR_LRM_OP_NOT_CONNECTED`
or something like that.  It would also partially explain the magic
constant present otherwise.

Reply:
Looking at other commits, rather `FEATURE_SET_MIN_FOR_NEW_LRM_OP_ERRORS`.

Reply:
Hmm, I'm not sure. The purpose of the feature set is to be a boundary point in behavior, so hopefully it's obvious that's what the comparison is for. Also, one feature set bump can cover multiple unrelated changes, so the constants could proliferate quickly. Perhaps we could describe each bump in the comment by the feature set definition.

Reply:
Either would work, both requiring consulting "place of use"
+ "place of explanation", i.e. two locations.

For the former, even slightly convoluted notation would be thinkable
when cooperated with the comparison function, like:

```
#define ENCODE_VER(major, minor, patch) \
	G_STATIC_ASSERT_EXPR(major >= 0 && major <= 99 \
                             minor >= 0 && minor <= 99 \
                             patch >= 0 && patch <= 99) \
	G_PASTE(major, G_PASTE(minor, patch))
  
#define FEATURE_SET_MIN_FOR_NEW_LRM_OP_ERRORS  ENCODE_VER(3, 2, 0)
```

At the place of use, exact numeric value is not that imporant as
the semantic guidance, IMHO.

And it would also alleviate computational complexity slightly
(parsing static string back to numbers is wasteful).

Reply:
Moreover, when the symbolic constants are deemed private,
`ENCODE_VER` can initially be:

```
#define ENCODE_VER(major, minor, micro) \
	G_STRINGIFY(major) "." G_STRINGIFY(minor) "." G_STRINGIFY(micro)
```

and said optimizations can arrive any time later (i.e., no change to the
comparison itself for the time being).